Dockerfile

```diff
@@ -2,7 +2,13 @@ FROM python:3.5.1-slim
 WORKDIR /app
 RUN groupadd --gid 1001 app && useradd -g app --uid 1001 --shell /usr/sbin/nologin app
 RUN apt-get update && apt-get install -y --no-install-recommends \
-    gcc libpq-dev
+    gcc libpq-dev curl apt-transport-https
+
+# Install node from NodeSource
+RUN curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
+    echo 'deb https://deb.nodesource.com/node_5.x jessie main' > /etc/apt/sources.list.d/nodesource.list && \
+    echo 'deb-src https://deb.nodesource.com/node_5.x jessie main' >> /etc/apt/sources.list.d/nodesource.list && \
+    apt-get update && apt-get install -y nodejs
```

Comment:
This causes us to run apt-get update and install twice. The curl and echo stuff should be moved above line 4, and the install nodejs bit should be on line 5.